src/uag.c

```diff
@@ -677,17 +661,18 @@ int uag_reset_transp(bool reg, bool reinvite)
 		}
 
 		/* update all active calls */
-		if (reinvite) {
-			struct le *lec;
+		if (!reinvite)
+			continue;
 
-			for (lec = ua_calls(ua)->head; lec; lec = lec->next) {
-				struct call *call = lec->data;
-				const struct sa *laddr;
+		for (lec = ua_calls(ua)->head; lec; lec = lec->next) {
+			struct call *call = lec->data;
+			struct sa laddr;
 
-				laddr = net_laddr_af(net, call_af(call));
+			if (net_dst_source_addr_get(stream_raddr(audio_strm(
+						call_audio(call))), &laddr))
+				continue;
```

Comment:
this code uses the audio stream address.

please note that in SDP the audio and video medialines can have different IP-addresses
example:

audio: 1.2.3.4
video: 5.6.7.8

Reply:
Baresip/re puts only one laddr into SDP. So we have to make a decision.